lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -9,7 +9,6 @@
 from django.contrib.sites.shortcuts import get_current_site
```

Comment:
I am not sure why we made changes in this file?

Reply:
Because a lot of code was repetitive so I added some utilities which could be re-used at both endpoints